### DISCUSSION TITLE
how to distinguish the two groups of the Status Bar?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#370] [Extension Development QnA] [answered]
When I read the document of VSCode Extension:

>[Status Bar](https://code.visualstudio.com/api/ux-guidelines/overview#status-bar)
The [Status Bar](https://code.visualstudio.com/api/ux-guidelines/status-bar) provides contextual information about the workspace and currently active file. It renders two groups of [Status Bar Items](https://code.visualstudio.com/api/ux-guidelines/status-bar#status-bar-items).

I want to know the two groups about the Status Bar.
![image](https://user-images.githubusercontent.com/12727582/210755217-c8cf00c0-b4e5-46a9-8bd1-8e9b9731f84d.png)

how to distinguish the two groups of it?

**Comment by gjsjohnmurray** [comment, marked as answer]
I think it is referring to the left-aligned group and the right-aligned group.

**Comment by Muhammadahmed23** [comment]
Hi,

Dear liaodalin19903,

Here['](https://gulfcareerz.info/)s a solution, To distinguish the two groups of the Status Bar, follow these steps:

1. **Look for visual cues**: The Status Bar typically has two distinct sections, often separated by a line or a different color scheme. Pay attention to any visual cues that indicate the division between the groups.
2. **Check for different icons or indicators**: Each group may have its own set of icons or indicators. For example, one group might display network signal strength, battery level, and time, while the other group might show notifications or system icons. Look for differences in the icons or indicators used.
3. **Explore settings or customization options**: Some devices or operating systems provide options to customize the Status Bar. Check the settings or customization options to see if you can configure the appearance or arrangement of the two groups.
4. **Consult the device or operating system documentation**: If you're unsure about the specific details of the two groups in the Status Bar, consult the documentation or user guides provided by the device manufacturer or the operating system developer. They should provide more information on how the Status Bar is organized and what each group represents.

You should be able to distinguish the two groups in the Status Bar and understand their respective elements or functionalities.

**Reply by gjsjohnmurray** [reply]
@Muhammadahmed23 did you write that yourself?
